pom.xml

```diff
@@ -44,22 +44,32 @@
         <air.check.skip-spotbugs>true</air.check.skip-spotbugs>
         <air.check.skip-pmd>true</air.check.skip-pmd>
         <air.release.preparation-goals>clean verify -DskipTests</air.release.preparation-goals>
+        <dep.jetty.version>12.0.9</dep.jetty.version>
```

Comment:
Is this needed? This is a default version in latest airlift. Please drop it.

Reply:
Please correct me if I'm wrong.
`<dep.jetty.version>` is being defined in the root pom.xml in Airlift.
It doesn't exist in `io.airlift:bom` nor `io.airlift:airbase`, so I cannot reuse it.

Reply:
you can test that ..  but we can also sort that out in a follow up